cpp/src/barretenberg/proof_system/composer/composer_helper_lib.hpp

```diff
@@ -1,179 +0,0 @@
-#pragma once
```

Comment:
This file was an exact copy of composer_lib and was not used at all so I've deleted it entirely.